### PR TITLE
fix(PanoViewer): fix delayed frame is not render

### PR DESCRIPTION
**Pull request by jongmoon**
## Issue
Ref #282

### CODE REVIEW

File: src/PanoViewer/PanoViewer.js [hunk 1]
```diff
@@ -801,26 +802,71 @@ export default class PanoViewer extends Component {
...
+	 * So browser cannot render the frame and may be freezing.
+	 */
+	_renderLoopForNextTick() {
+		const before = performance.now();
+
+		this._render();
+
+		const diff = performance.now() - before;
+
+		if (this._rafTimer) {
```
Comment (younkue) [inline src/PanoViewer/PanoViewer.js]: @jongmoon 
I think the _renderLoop > _render process happens sequentially. Is there any clearTimeout?

File: src/utils/browser.js [hunk 2]
```diff
@@ -7,8 +7,13 @@ const win = typeof window !== "undefined" && window.Math === Math ? window : typ
 /* eslint-enable no-new-func, no-nested-ternary */
 
 const doc = win.document;
+const userAgent = win.navigator.userAgent;
+const IS_SAFARI_ON_DESKTOP = userAgent.indexOf("Safari") !== -1 &&
+														userAgent.indexOf("Chrome") === -1 &&
```
Comment (younkue) [inline src/utils/browser.js]: @jongmoon What's wrong with the indent?

File: src/utils/browser.js [hunk 3]
```diff
@@ -7,8 +7,13 @@ const win = typeof window !== "undefined" && window.Math === Math ? window : typ
...
+														userAgent.indexOf("Mac OS X") !== 1;
```
Comment (WoodNeck) [inline src/utils/browser.js]: Shouldn't it have to be `-1` not `1`?

Comment (jongmoon) [reply src/utils/browser.js]: @WoodNeck Thank you!!

File: src/utils/browser.js (hunk 2 above)
Comment (jongmoon) [reply src/utils/browser.js]: Tab has 8 spaces on Github but on my IDE(VS Code), Tab has 2 spaces.
These makes difference.

**Pull request merged**